### PR TITLE
added some shield and armor definitions to glossary

### PR DESCRIPTION
**Pull request by bleehu**
I'm holding off on Armor Strength/Armor Points until it has been updated in Basic Rules and the 05_armor_list documents.

### CODE REVIEW

**Comment by bleehu** [comment]
This should satisfy almost all of Issue #300

File: 03_Glossary_of_Terms.txt [hunk 1]
```diff
@@ -11,6 +11,13 @@ it does in English. Instead, an Action is a measure of time specifically during
 a Player's turn in Combat. See "Actions" Section in the "Combat" Chapter in 
 Basic Rules.txt.
 
+Armor Piercing Level:
+    Different sets of Armor stop different amounts of firepower depending on
```
Comment (Turtlelord26) [inline 03_Glossary_of_Terms.txt:15]: Nonblocker but I would prefer the more precise term 'penetration' over the general and not well-defined 'firepower'.

File: 03_Glossary_of_Terms.txt [hunk 2]
```diff
@@ -65,11 +72,22 @@ how the NPCs behave and act.
 GM: 
     See Game Master.
 
+Hardpoint:
+    An Armor Hardpoint is a purpose-built universal modular slot that advanced
```
Comment (Turtlelord26) [inline 03_Glossary_of_Terms.txt]: Universal and modular are redundant. Also grammar requires separation of adjectives with commas, 
e.x. purpose-built, modular slot

File: 03_Glossary_of_Terms.txt [hunk 3]
```diff
@@ -126,12 +140,21 @@ Reactions:
 Reactions occur at the immediate moment that they are triggered. See "Reactions 
 & Held Actions" in the basic rules document.
 
+    In brief, Proficiency is a sufficient aptitude such that a character can use 
+a given type of Weapon or Armor in combat. Proficiency can also be the ability 
+to use a Skill with given capability.
```
Comment (Turtlelord26) [inline 03_Glossary_of_Terms.txt]: Should this be up with the rest of Proficiency instead of below Reactions?

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]

**Comment by bleehu** [comment]
How's that?

**Review by Turtlelord26 — APPROVED** [review]

**Pull request merged**